db/seeds.rb

```diff
@@ -1,12 +1,34 @@
 if Color.table_exists? && Color.count == 0
-  Color.create name: 'red',   hex_value: '#FF0000'
-  Color.create name: 'green', hex_value: '#00FF00'
-  Color.create name: 'blue',  hex_value: '#0000FF'
+  Color.create name: 'red',    hex_value: '#FF0000'
+  Color.create name: 'green',  hex_value: '#00FF00'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.